### ISSUE TITLE
Shortcut refinement error produced by IDE only

### ISSUE THREAD

**Issue by ASzc** [ceylon/ceylon-ide-eclipse#1183]
Using Eclipse Luna with Ceylon IDE plugin Version `1.1.0.v20141013-1438-Final`, the following minimal module always produces this error: `formal member 'tag' of 'Node' not implemented in class hierarchy  run.ceylon  line 11` (at the definition of `ProperCharsetMeta`)

The command line tools will always build and run the module without issue.

```ceylon
==> ./source/mwe/module.ceylon <==
module mwe "0.1" {
    shared import ceylon.html "1.1.0";
}

==> ./source/mwe/package.ceylon <==
shared package mwe;

==> ./source/mwe/run.ceylon <==
import ceylon.html {
    html5,
    Html,
    Head,
    Meta
}
import ceylon.html.serializer {
    consoleSerializer
}

shared class ProperCharsetMeta(charset = "utf-8") extends Meta("") {
    shared String charset;
    shared actual default [<String->Object>*] attributes => [
        "charset"->charset
    ];
}

shared void run() {
    Html html = Html {
        html5;
        Head {
            ProperCharsetMeta()
        };
    };
    consoleSerializer.serialize(html);
}
```

If you force the IDE to execute the `run` function, the console output shows:

```
Warning: Failed to define class mwe.ProperCharsetMeta in Module "mwe:0.1" from Ceylon ModuleLoader: RootRepositoryManager: FileContentStore: /home/alex/.ceylon/cache java.lang.VerifyError: class mwe.ProperCharsetMeta overrides final method getTag.()Lceylon/html/Tag;
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.redhat.ceylon.launcher.Launcher.runInJava7Checked(Launcher.java:104)
	at com.redhat.ceylon.launcher.Launcher.run(Launcher.java:37)
	at com.redhat.ceylon.launcher.Launcher.run(Launcher.java:30)
	at com.redhat.ceylon.launcher.Launcher.main(Launcher.java:23)
Caused by: java.lang.VerifyError: class mwe.ProperCharsetMeta overrides final method getTag.()Lceylon/html/Tag;
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:760)
	at org.jboss.modules.ModuleClassLoader.doDefineOrLoadClass(ModuleClassLoader.java:361)
	at org.jboss.modules.ModuleClassLoader.defineClass(ModuleClassLoader.java:482)
	at org.jboss.modules.ModuleClassLoader.loadClassLocal(ModuleClassLoader.java:277)
	at org.jboss.modules.ModuleClassLoader$1.loadClassLocal(ModuleClassLoader.java:92)
	at org.jboss.modules.Module.loadModuleClass(Module.java:568)
	at org.jboss.modules.ModuleClassLoader.findClass(ModuleClassLoader.java:205)
	at org.jboss.modules.ConcurrentClassLoader.performLoadClassUnchecked(ConcurrentClassLoader.java:459)
	at org.jboss.modules.ConcurrentClassLoader.performLoadClassChecked(ConcurrentClassLoader.java:408)
	at org.jboss.modules.ConcurrentClassLoader.performLoadClass(ConcurrentClassLoader.java:389)
	at org.jboss.modules.ConcurrentClassLoader.loadClass(ConcurrentClassLoader.java:134)
	at mwe.run_.run(run.ceylon:22)
	at mwe.run_.main(run.ceylon)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at ceylon.modules.api.runtime.SecurityActions.invokeRunInternal(SecurityActions.java:58)
	at ceylon.modules.api.runtime.SecurityActions.invokeRun(SecurityActions.java:48)
	at ceylon.modules.api.runtime.AbstractRuntime.invokeRun(AbstractRuntime.java:101)
	at ceylon.modules.api.runtime.AbstractRuntime.execute(AbstractRuntime.java:167)
	at ceylon.modules.api.runtime.AbstractRuntime.execute(AbstractRuntime.java:151)
	at ceylon.modules.Main.execute(Main.java:69)
	at ceylon.modules.Main.main(Main.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.jboss.modules.Module.run(Module.java:312)
	at org.jboss.modules.Main.main(Main.java:460)
	at ceylon.modules.bootstrap.CeylonRunTool.run(CeylonRunTool.java:208)
	at com.redhat.ceylon.common.tools.CeylonTool.run(CeylonTool.java:352)
	at com.redhat.ceylon.common.tools.CeylonTool.execute(CeylonTool.java:291)
	... 8 more
```

It seemed that this could be a regression of #758, however the code there works fine for me.

**Comment by davidfestal** [comment]
@ASzc : Could you please confirm that it's also fixed for you  ?

**Comment by ASzc** [comment]
Yes, this is fixed for me in plugin version `1.1.1.v20150627-1247-Final`. Thanks.
